src/umpire/ResourceManager.cpp

```diff
@@ -95,6 +109,16 @@ ResourceManager::ResourceManager() :
     std::make_shared<resource::RocmPinnedMemoryResourceFactory>());
 #endif
 
+#if defined(UMPIRE_ENABLE_DEVICE) and defined(UMPIRE_ENABLE_NUMA)
```

Comment:
Does `and` work here rather than `&&`?

Reply:
I think this works, but for uniformity I'll change it to &&